filebeat/module/pensando/dfw/_meta/fields.yml

```diff
@@ -0,0 +1,59 @@
+- name: payload_raw
```

Comment:
this seems to be removed at https://github.com/elastic/beats/pull/21063/files#diff-6aa985f65f93a9205435ffd14f57f3cc69f7acf9df33ac4f188fc4397c071520R210

Reply:
Changed this so it is not removed